addons/mail/controllers/message_reaction.py

```diff
@@ -5,13 +5,23 @@
 from odoo import http
 from odoo.http import request
 from odoo.addons.mail.models.discuss.mail_guest import add_guest_to_context
+from odoo.addons.mail.tools.discuss import Store
 
 
 class MessageReactionController(http.Controller):
     @http.route("/mail/message/reaction", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
-    def mail_message_add_reaction(self, message_id, content, action):
+    def mail_message_add_reaction(self, message_id, content, action, **kwargs):
```

Comment:
While we are at it, this method should be renamed, as it is no longer just "add" reaction, it can also do "remove" depending on action.